src/Map.jsx

```diff
@@ -75,7 +75,21 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  getRouteBetweenStops(route) {
```

Comment:
@EddyIonescu 
The point of the function is two get the route info (all the stops between the  two included stops)
We need these points so we can  (hopefully) draw a more accurate line over the map
My concern is though that the sub segment line we draw goes over the road and not the stops themselves

Reply:
I see, I was thinking of snapping each of the two stops to the route's line and then taking a subsection of the line?